### PR TITLE
Traduciendo código fuente al español

### PR DESCRIPTION
**Pull request by raydelto**
Traduciendo gran parte del código fuente de la sección de estructuras de decisión al español

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/spanish/cplusplus/if-else-statement/index.md [hunk 1]
```diff
@@ -1,192 +1,193 @@
...
+```
+Introduzca un número: 
+ -200 
+ Usted ha introducido un número negativo: -200 
+ Este mensaje siempre se muestra al ejecutarse el programa.
+```
+
+[Prueba el código tu mismo](https://repl.it/MzBq)
+
+# **No dude en preguntar cualquier duda en la página de GitHub de [FreeCodeCamp](https://forum.freecodecamp.org/) o en [el foro de FreeCodeCamp.](https://forum.freecodecamp.org/)**
```
Comment (lynxlynxlynx) [inline guide/spanish/cplusplus/if-else-statement/index.md]: why is this a title? It looks very odd!

File: guide/spanish/cplusplus/if-else-statement/index.md [hunk 2]
```diff
@@ -1,192 +1,193 @@
...
+```
+
+#### Salida
+```
+Enter first number: 
+ 85 
+ Enter second number: 
+ 86 
+ Number 2 is greater. 
+```
```
Comment (lynxlynxlynx) [inline guide/spanish/cplusplus/if-else-statement/index.md]: This and the previous example program (and output) are missing translations.

**Review by lynxlynxlynx — COMMENTED** [review]
fixes headmatter and adds some translations, however the other half of the article is still only partly ok.

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**